tests/tests_pytorch/core/test_lightning_module.py

```diff
@@ -425,6 +425,17 @@ def test_proper_refcount():
     assert sys.getrefcount(torch_module) == sys.getrefcount(lightning_module)
 
 
+def test_lightning_module_scriptable():
```

Comment:
nit: this would have fit better in `tests/tests_pytorch/models/test_torchscript.py`